airflow/executors/kubernetes_executor.py

```diff
@@ -256,15 +256,15 @@ class KubernetesJobWatcher(multiprocessing.Process, LoggingMixin):
     def __init__(self,
                  namespace: str,
                  watcher_queue: 'Queue[KubernetesWatchType]',
-                 resource_version: str,
+                 resource_version: Optional[str],
                  worker_uuid: Optional[str],
                  kube_config: Configuration):
         multiprocessing.Process.__init__(self)
-        self.namespace: str = namespace
```

Comment:
Why are we removing this?

Reply:
Not removing - just removing the types. The types are inferred from the __init__ parameters so not needed to be duplicated.